docs/overview.md

```diff
@@ -0,0 +1,135 @@
+## Safe Protocol
+
+The Safe Protocol is a non-custodial set of smart contracts that allows users to create programmable multi-signature accounts that require multiple parties to authorize transactions. This provides an added layer of security and reduces the risk of funds being lost or stolen compared to regular EOA account.
+
+### Basic flows:
+
+**Deploying a Safe account**
+
+-   A user creates a Safe account by deploying a Proxy contract pointing to the Safe implementation contract.
```

Comment:
Maybe we can mention that the Safe contract is agnostic to the proxy implementation (and reference a couple).

Reply:
Would mention but referencing is a little bit out of scope of the basic flow imo